purchases/src/test/java/com/revenuecat/purchases/backend_integration_tests/LoadShedderBackendIntegrationTest.kt

```diff
@@ -155,52 +154,4 @@ internal class LoadShedderBackendIntegrationTest: BaseBackendIntegrationTest() {
         }
         assertSigningPerformed()
     }
-
-    @Test
-    fun `can perform login backend request`() {
```

Comment:
I've removed the logIn tests from the load shedder integration tests since it's not actually supported. It used to work since we weren't actually pointing to the load shedder. The behavior is still tested in production in `ProductionBackendIntegrationTest`